src/Client/Clients/TextDocumentClient.Completions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using OmniSharp.Extensions.LanguageServer.Models;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OmniSharp.Extensions.LanguageServerProtocol.Client.Clients
+{
+    using Utilities;
```

Comment:
_nit_: Not a huge fan of nested usings.

Reply:
No problem; I tend to use it as a reminder of which namespaces are from "my" code vs external code, but I'm happy to change this over. My rule is usually "match the style already used by the project" :)